src/main/java/com/blogify/blogapi/repository/model/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.blogify.blogapi.repository.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.*;
+import lombok.Data;
+
+@Entity
+@Table(name = "comments")
+@Data
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Use an UUID instead of Long and delete GeneratedValue because ids are provided by client